lib/assert/element.js

```diff
@@ -0,0 +1,176 @@
+'use strict';
+
+var util = require('util');
+
+var assert = require('assertive');
+var _ = require('lodash');
+
+function isTextOrRegexp(textOrRegExp) {
+  return _.isString(textOrRegExp) || _.isRegExp(textOrRegExp);
+}
+
+exports._getElementWithProperty = function _getElementWithProperty(selector, property) {
+  var element = this._getElement(selector);
+  return [ element, element.get(property) ];
+};
```

Comment:
FWIW, this method seems weird and redundant, and makes the code harder to read.  The consumers that use it refer to the element and its property value as `result[0]` and `result[1]`.  You could destructure but I feel like you should just inline.